src/config/dynamic-theme-fixes.config

```diff
@@ -6848,6 +6848,13 @@ INVERT
 
 ================================
 
+crowdfense.com
+
+INVERT
+.logo
+
```

Comment:
Can you add `.mega-menu.menu-hover-line > li.menu-item > a::before` too for the line above upper menu?

I wanted to add it by myself but edit option is disabled for this PR for me for some reason.